Content.Server/Antag/Components/AntagSelectionComponent.cs

```diff
@@ -154,6 +155,12 @@ public partial struct AntagSelectionDefinition()
     [DataField]
     public ProtoId<StartingGearPrototype>? StartingGear;
 
+    /// <summary>
+    /// A list of role loadouts to be equipped to the player.
+    /// </summary>
+    [DataField]
+    public List<ProtoId<RoleLoadoutPrototype>>? RoleLoadout;
+
```

Comment:
Is there even a reason to have startinggear anymore at that point considering they can contain multiple startinggears?

Also having a list doesn't make sense here unless it's randomly selected.

Reply:
Strictly speaking, no we don't need startinggear on this component anymore. We could put the existing startinggear prototypes inside loadoutgroups. Downstreams might still need startinggear, for all I know. Do you want me to remove startinggear from this component and just put it in breaking changes?

There is in fact a random selection for the roleloadout list, I just commented it poorly. I will change that now

Reply:
I did the thing. I will phase out using startingGear on a separate PR as its kind of a big delta